### ISSUE TITLE
Create an issue tracker for new leagues or open up Discord to the public.

### ISSUE THREAD

**Issue by emirkmo** [PathOfBuildingCommunity/PathOfBuilding#3283]
### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### Is your feature request related to a problem?

I started looking at implementing some of the new patch changes when I saw random PRs being opened for the same thing already. Seems like there is a discord where people are organizing. This makes it hard for a newcomer to come and contribute as I don't know what's being worked on, or not.

### Describe the solution you'd like

So I understand that it may be a lot of work to open up an issue for each new PR before that PR is being worked on, (although I would recommend it), but atleast for new leagues there should be an issue to track a master list of things to do before the new patch, or make 1 of the channels on the discord public so people can drive-by contribute. 

### Describe alternatives you've considered

Master issue tracker, either on Github or website or Discord.

Opening up public discord.

Requiring an issue to be written before work on a PR begins.

Creating a list of contributable issues for each new League, (although this kind of exists, it's not for bringing in new stuff from the patch, but for old issues or bugs).

**Comment by Aradiv** [comment]
CONTRIBUTING.md

> Before submitting a pull request:
> Familiarise yourself with the code base here to get you started.
> There is a Discord server for active development on the fork and members are happy to answer your questions there. If you are interested in joining, send a private message to any of Cinnabarit#1341, LocalIdentity#9871, Yamin#5575 and we'll send you an invitation.

**Comment by emirkmo** [comment]
Yes I’ve read that. It involves reaching out to people, waiting for them to get online, then possibly getting an invite. I was on a train and wanted to see if I could spend a few hours to help. Anyway, if it’s not of interest, this suggestion is certainly nothing major and nothing worth spending more time on. It’s just a way to lower the barrier to getting more contributions.
Cheers!

**Comment by pHiney** [comment]
The other thing needed if issues are created per task, is the ability say they are assigned. 
The discord method is possibly the simplest.
They may have a shared spreadsheet also (eg: googledocs)

pH
